support/cas-server-support-cassandra-ticket-registry/src/main/java/org/apereo/cas/config/CassandraTicketRegistryTicketCatalogConfiguration.java

```diff
@@ -0,0 +1,70 @@
+package org.apereo.cas.config;
+
+import org.apereo.cas.configuration.CasConfigurationProperties;
+import org.apereo.cas.ticket.TicketCatalog;
+import org.apereo.cas.ticket.TicketDefinition;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * This is {@link CassandraTicketRegistryTicketCatalogConfiguration}.
+ *
+ * @author David Rodriguez
+ * @since 5.2.0
+ */
+@Configuration("cassandraTicketRegistryTicketMetadataCatalogConfiguration")
+@EnableConfigurationProperties(CasConfigurationProperties.class)
+public class CassandraTicketRegistryTicketCatalogConfiguration extends CasCoreTicketCatalogConfiguration {
+
+    /**
+     * Name for the ticketGrantingTicket tickets table.
+     */
+    public static final String TGT_TABLE = "ticketGrantingTicket";
```

Comment:
These constants all need to be made private. Nobody outside of this class should know anything about the tables, etc.